src/preact/bento-ce.js

```diff
@@ -115,3 +72,65 @@ if (typeof AMP !== 'undefined' && AMP.BaseElement) {
 }
 
 export {BaseElement};
+
+/** @type {typeof HTMLElement} */
+let ExtendableHTMLElement;
+/** @type {Window} */
+let win;
+
+/**
+ * @param {typeof import('./base-element').PreactBaseElement} BaseElement
+ * @param {Window} _win
+ * @return {typeof HTMLElement}
+ */
+function createBentoElementClass(BaseElement, _win = self) {
+  if (!ExtendableHTMLElement || win !== _win) {
+    win = _win;
+    ExtendableHTMLElement = maybeWrapNativeSuper(
+      // @ts-ignore
```

Comment:
I don't think we should need the ts-ignore.
I think in this case you can use `devAssert(win)` to inform below code it isn't null. That or use `_win`

Reply:
i wasn't ignoring a null error, I was ignoring this:

`Property 'HTMLElement' does not exist on type 'Window'.`

because HTMLElement doesn't exist on `window`'s type?.

I was able to get around this by doing this:

```
/** @type {typeof self} */
let win;
...
ExtendableHTMLElement = maybeWrapNativeSuper(win.HTMLElement);
```

Apparently `HTMLElement` does exist on `self`'s type, so I don't get the same error.

Reply:
Can you create a new file: `src/preact/global.d.ts` with the contents:
```
export {};

declare global {
  interface Window {
    HTMLElement: typeof HTMLElement;
  }
}
```

Hopefully we can figure out something better later.

Reply:
resolved in dms

we're going to use `typeof globalThis` as the type for `win`